server/assets/images/logo.svg

```diff
@@ -1,13 +1,176 @@
+	.st108{clip-path:url(#SVGID_4_);fill:#F5A42E;}
+	.st109{clip-path:url(#SVGID_4_);fill:#F5A42D;}
+	.st110{clip-path:url(#SVGID_4_);fill:#F5A52D;}
+	.st111{clip-path:url(#SVGID_4_);fill:#F5A62C;}
+	.st112{clip-path:url(#SVGID_4_);fill:#F5A72C;}
+	.st113{clip-path:url(#SVGID_4_);fill:#F5A82B;}
+	.st114{clip-path:url(#SVGID_4_);fill:#F6A82A;}
+	.st115{clip-path:url(#SVGID_4_);fill:#F6A92A;}
+	.st116{clip-path:url(#SVGID_4_);fill:#F6AA29;}
+	.st117{clip-path:url(#SVGID_4_);fill:#F6AA28;}
+	.st118{clip-path:url(#SVGID_4_);fill:#F6AB28;}
+	.st119{clip-path:url(#SVGID_4_);fill:#F6AC27;}
+	.st120{clip-path:url(#SVGID_4_);fill:#F6AD26;}
+	.st121{clip-path:url(#SVGID_4_);fill:#F7AE25;}
+	.st122{clip-path:url(#SVGID_4_);fill:#F7AF24;}
+	.st123{clip-path:url(#SVGID_4_);fill:#F7B023;}
+	.st124{clip-path:url(#SVGID_4_);fill:#F7B022;}
+	.st125{clip-path:url(#SVGID_4_);fill:#F7B122;}
+	.st126{clip-path:url(#SVGID_4_);fill:#F7B221;}
+	.st127{clip-path:url(#SVGID_4_);fill:#F7B320;}
+	.st128{clip-path:url(#SVGID_4_);fill:#F7B31F;}
+	.st129{clip-path:url(#SVGID_4_);fill:#F8B41E;}
+	.st130{clip-path:url(#SVGID_4_);fill:#F8B51D;}
+	.st131{clip-path:url(#SVGID_4_);fill:#F8B51C;}
+	.st132{clip-path:url(#SVGID_4_);fill:#F8B61B;}
+	.st133{clip-path:url(#SVGID_4_);fill:#F8B71A;}
+	.st134{clip-path:url(#SVGID_4_);fill:#F8B718;}
+	.st135{clip-path:url(#SVGID_4_);fill:#F8B817;}
+	.st136{clip-path:url(#SVGID_4_);fill:#F9B916;}
+	.st137{clip-path:url(#SVGID_4_);fill:#F9BA14;}
+	.st138{clip-path:url(#SVGID_4_);fill:#F9BA13;}
+	.st139{clip-path:url(#SVGID_4_);fill:#F9BB11;}
+	.st140{clip-path:url(#SVGID_4_);fill:#F9BC0F;}
+	.st141{clip-path:url(#SVGID_4_);fill:#F9BC0D;}
+	.st142{clip-path:url(#SVGID_4_);fill:#F9BD0B;}
+	.st143{clip-path:url(#SVGID_4_);fill:#F9BE09;}
+	.st144{clip-path:url(#SVGID_4_);fill:#FABF06;}
+	.st145{clip-path:url(#SVGID_4_);fill:#FABF04;}
+	.st146{fill:#87CCCC;}
+	.st147{fill:#3D3D3B;}
+	.st148{fill:#E83A42;}
+	.st149{fill:url(#SVGID_5_);}
+	.st150{fill:#FFFFFF;}
+	.st151{fill:#44B69F;}
+	.st152{fill:#646363;}
+	.st153{opacity:0.4;}
+	.st154{fill:#D0D0D0;}
+	.st155{fill:none;stroke:#646363;stroke-width:21;stroke-miterlimit:10;}
+	.st156{fill:url(#SVGID_6_);}
+	.st157{fill:url(#SVGID_7_);}
+	.st158{fill:none;stroke:#D0D0D0;stroke-width:3;stroke-linecap:square;stroke-miterlimit:10;}
+	.st159{fill:none;stroke:#646363;stroke-width:3;stroke-linecap:square;stroke-miterlimit:10;}
+</style>
+<g>
+	<linearGradient id="SVGID_2_" gradientUnits="userSpaceOnUse" x1="163.1266" y1="133.72" x2="13.0306" y2="7.7745">
+		<stop  offset="0" style="stop-color:#E83A47"/>
+		<stop  offset="1" style="stop-color:#FABF04"/>
+	</linearGradient>
+	<polygon style="fill:url(#SVGID_2_);" points="112,0 103.7,0 101,0 114.5,29.9 117.2,29.9 125.4,29.9 142.2,29.9 120.2,78.8 
+		84.6,0 70.3,0 51.9,0 43.6,0 35.4,0 0,0 0,151.7 29.9,151.7 29.9,90 75.9,90 62.5,60.1 29.9,60.1 29.9,29.9 48.9,29.9 57.1,29.9 
+		65.3,29.9 78.9,60.1 92.4,90 120.2,151.7 188.3,0 	"/>
+</g>
+</svg>
```

Comment:
This svg can be reduces by 5% with (https://www.svgminify.com/)

Reply:
SVG's are automatically compresses/optimized when building the script.